plugins/amazonq/chat/jetbrains-community/src/software/aws/toolkits/jetbrains/services/cwc/controller/ChatController.kt

```diff
@@ -103,6 +113,32 @@ class ChatController private constructor(
         authController = AuthController(),
     )
 
+    init {
```

Comment:
`ChatController` can be initialized multiple times in a given project instance

Reply:
It now only runs once. A flag in the companion object is there